src/App.js

```diff
@@ -1,17 +1,25 @@
 import React from 'react';
+import {
+  BrowserRouter as Router, Routes, Route,
+} from 'react-router-dom';
 import Calc from './components/calculator';
+import NotFoundPage from './components/ErrorPage';
+import Home from './components/Home';
+import Quotes from './components/Quotes';
+import Navigation from './components/Navigation';
 
-class App extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {};
-  }
-
-  render() {
-    return (
-      <Calc />
-    );
-  }
+function App() {
+  return (
+    <Router>
+      <Navigation />
+      <Routes>
+        <Route path="/Home" element={<Home />} />
+        <Route path="/calculator" element={<Calc />} />
+        <Route path="/Quotes" element={<Quotes />} />
+        <Route path="*" element={<NotFoundPage />} />
```

Comment:
- [ ] this is the second part of the issue I mentioned in the main comment, your `Home` element should be your root path which looks like below
```<Route path="/" element={<Home />} />```